webapp/src/modules/bid/actions.ts

```diff
@@ -84,9 +86,22 @@ export const acceptBidSuccess = (bid: Bid, txHash: string) =>
 export const acceptBidFailure = (bid: Bid, error: string) =>
   action(ACCEPT_BID_FAILURE, { bid, error })
 
+export const acceptBidtransactionSubmitted = (bid: Bid, txHash: string) =>
+  action(
+    ACCEPT_BID_TRANSACTION_SUBMITTED,
```

Comment:
We should add bid here to use it later in the track file:
```suggestion
    ACCEPT_BID_TRANSACTION_SUBMITTED,
    bid,
```